test/completion.js

```diff
@@ -38,4 +38,17 @@ describe('flag: --completion', function() {
     }
   });
 
+  it('shows error message for missing completion type', function(done) {
+    runner({ verbose: false })
+      .gulp('--completion')
+      .run(cb);
+
+    function cb(err) {
+      expect(function() {
+        throw err;
```

Comment:
Why would that match?  Maybe you are using the wrong assertion.  Also, you probably need to match `err.message`